plugins/arDominionB5Plugin/js/hierarchy.js

```diff
@@ -0,0 +1,162 @@
+"use strict";
+
+(function ($) {
+
+  $(function()
+  {
+    var $node = $('.browse.hierarchy');
+
+    if ($node.length)
+    {
+      $(loadTreeView);
+    }
```

Comment:
In this case this is the only change to the entire JS file, and I think it's safe enough to do it in the current one and avoid the duplication.

Reply:
Same as above - some additional changes with respect to deprecated jquery calls. Tested in the old theme and functionality is still present.  Moving to the root /js folder.